### DISCUSSION TITLE
Support for regression lines in XY Chart

### DISCUSSION THREAD

**Discussion by hafeja** [grafana/grafana#63173] [Dashboards, Panels & Visualizations]
Hey!

As user of plugin [XY Chart](https://grafana.com/grafana/plugins/xychart/) I'd like to request support for **regression lines** for this plugin.

Finally, a regression line should look e.g. like this:
![image](https://user-images.githubusercontent.com/59864250/217755914-29e9cda1-947c-4ef2-abdf-679fb7fdaedc.png)

**Expected Benefit**
A regression line transports the key message of a scatter plot to dashboard viewers **at a glance**.

It would be good to support different kinds of regression lines (Linear, Exponential, Power, ...) similar to the ones provided by the plugin [Scatter](https://grafana.com/grafana/plugins/michaeldmoore-scatter-panel/):

![image](https://user-images.githubusercontent.com/59864250/217756309-db7f60bd-2033-4cd1-adf3-b3f44ed6baca.png)
![image](https://user-images.githubusercontent.com/59864250/217756463-e4cd5a6e-bdf1-4d00-ab1e-a8231da7e731.png)

The **icing on the cake** would be the possibility for the dashboard creator to define the formula that is used to calculate the regression line.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82857. Feel free to continue the discussion around this there. Thank you!
